### PR TITLE
angular-google-analytics improvements

### PR DESCRIPTION
**Pull request by slawiko**
Issue #12229 

case 2. Improvement to existing type definition.
- Removes useless angular-google-analytics-service.d.ts file and adds this service in index.d.ts;
- Edits namespace usage (like in angular.ui.router and other packages). Previous version had a wrong (useless) [namespace declaration](https://www.typescriptlang.org/docs/handbook/namespaces-and-modules.html).

### CODE REVIEW

**Comment by slawiko** [comment]
@RyanCavanaugh maybe you can review it?

**Comment by vvakame** [comment]
*angular-google-analytics/angular-google-analytics-service.d.ts*

REMOVED!
------

*angular-google-analytics/index.d.ts*

to author (@cyrilschumacher). Could you review this PR?
:+1: or :-1:?

Checklist

* [ ] pass the Travis CI test?

File: angular-google-analytics/index.d.ts [hunk 1]
```diff
@@ -4,171 +4,232 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference types="angular" />
-/// <reference path="angular-google-analytics-service.d.ts" />
+import * as angular from 'angular';
 
-declare namespace angular.google.analytics {
+declare module 'angular' {
```
Comment (vvakame) [inline angular-google-analytics/index.d.ts]: is this right?

Comment (slawiko) [inline angular-google-analytics/index.d.ts]: As I understand - yes. The easiest way it's compare with existed types e.g. [angular-translate](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/types-2.0/angular-translate/index.d.ts), [angulat-ui-bootstrap](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/types-2.0/angular-ui-bootstrap/index.d.ts) or [angular-ui-router](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/types-2.0/angular-ui-router/index.d.ts).

**Pull request merged**